acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/signing/ValueTransferAcceptanceTest.java

```diff
@@ -35,7 +35,7 @@
 public class ValueTransferAcceptanceTest extends AcceptanceTestBase {
 
   private static final String RECIPIENT = "0x1b00ba00ca00bb00aa00bc00be00ac00ca00da00";
-  private static final long FIFTY_TRANSACTIONS = 50;
+  private static final long NO_OF_TRANSACTIONS = 50;
```

Comment:
Change this back, it's a constant for the number 50, which is being used as 50 transaction ...NO_OF_TRANSACTIONS provide the same meaning when reading the code that uses the constant

Reply:
I haven't changed that .... Will do!